libs/core/src/lib/rating-indicator/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const INDICATOR_PREFIX = 'fd-rating-indicator';
+
+export const INDICATOR_CLASSES = {
+    halves: `${INDICATOR_PREFIX}--half-star`,
+    icon: `${INDICATOR_PREFIX}--icon`,
+    hideDynamicText: `${INDICATOR_PREFIX}--hide-dynamic-text`
+};
+export const INDICATOR_DEFAULT_TOTAL = 5;
+export const INDICATOR_RANGE = {
```

Comment:
This range is specified in the docs?

Reply:
Nope, same as not specified ngModel and any control details for this component

Reply:
Then I'm not sure if we should specify some fixed boundaries for the capacity of the Rating Indicator. I think its good to have this, but `<3;7>` is quite narrow. I will bring this to one of the meetings to see what the rest of the team thinks.

Reply:
@mykolavorobiov We had a conversation on this topic and we decided not to specify the boundaries for the loading indicator capacity.